requirements/edx/kernel.in

```diff
@@ -72,6 +72,7 @@ edx-codejail>=4.0.0
 edx-django-utils>=5.14.1             # Utilities for cache, monitoring, and plugins
 edx-drf-extensions
 edx-enterprise
+enterprise-integrated-channels      # Integrated Channels to transmit content metadata and learner data to enterprise customers
```

Comment:
I think we might want to put this in `bundled.in` instead -- can we try that? (From looking at the file comments, I think the idea is that we should put things into that file instead of `kernel.in` as long as that won't cause tests to fail.)